waspc/src/Wasp.hs

```diff
@@ -113,6 +117,11 @@ setApp wasp app = wasp { waspElements = (WaspElementApp app) : (filter (not . is
 fromApp :: App -> Wasp
 fromApp app = fromWaspElems [WaspElementApp app]
 
+-- * Routes
+
+getRoutes :: Wasp -> [Route]
+getRoutes wasp = [route | (WaspElementRoute route) <- waspElements wasp]
```

Comment:
Your arch enemy :D.

Reply:
ah yes! I learned about type lvl programming, thinking it might be the solution but it is not :D Next step is then TH/lens/prism. I shall defeat it :D